android/app/build.gradle

```diff
@@ -53,7 +53,7 @@ android {
     }
 
     kotlinOptions {
-        jvmTarget = '1.8'
+        jvmTarget = '11'
```

Comment:
Could you revert this? I also had some issues with this, will try to fix this in a version bump